doc/how-search-works.md

```diff
@@ -0,0 +1,50 @@
+# How Search Works
+
+Search API provides a `/search` endpoint for users to of the API to search
+for documents.
+
+You can also use `/batch_search` to send multiple queries.
+
+## Document retrieval
+
+On receiving a request to `/search` (a search query), Search API will parse
+the query, construct an Elasticsearch query, and then retrieve documents
+from Elasticsearch.
+
+## Relevancy
+
+If a search query requests that search results be ordered by relevance to the
+query, Search API will attempt to order the search results in the most
+relevant way possible.
```

Comment:
i'm not sure this paragraph adds much.  It's equally true for any other ordering.